src/main/java/frc/robot/commands/Score.java

```diff
@@ -27,6 +39,6 @@ public void end(boolean interrupted) {}
   // Returns true when the command should end.
   @Override
   public boolean isFinished() {
-    return false;
+    return cubePositioner.getHasObject();
```

Comment:
shouldn't it have an object right when this command starts? ie before we've scored the cube in our possession